content/en/database_monitoring/setup_mysql/rds.md

```diff
@@ -178,6 +178,12 @@ instances:
     port: 3306
     username: datadog
     password: '<YOUR_CHOSEN_PASSWORD>' # from the CREATE USER step earlier
+    
+    
+     # After adding your project and instance, configure the Datadog AWS integration to pull additional cloud data such as CPU, Memory, etc.
+    aws:
+      deployment_type: '<DEPLOYMENT_TYPE>'
+      name: '<AWS_INSTANCE_ENDPOINT>'
```

Comment:
This looks like the azure model for this. AWS [is a bit different](https://github.com/DataDog/integrations-core/blob/master/postgres/datadog_checks/postgres/data/conf.yaml.example#L373-L380):
```suggestion
    aws:
      instance_endpoint: '<AWS_INSTANCE_ENDPOINT>'
```